src/CarbonAware/src/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file should be added to the gitignore file (the path would not work for my code).